docker/ubuntu-vsc-base/24.04/entrypoint.sh

```
@@ -26,6 +26,89 @@ if [ "x${1:-}" = "x--version" ]; then
 	exit
 fi
 
-set -x
-exec "$@"
-x
+[ "${USER_NAME:-root}" != "root" ] || die "Invalid \$USER_NAME (${USER_NAME})"
+
+# helper to find existing group by GID
+find_group_by_gid() {
+	local name
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)